src/app/editor-toolbar/editor-toolbar-save/editor-toolbar-save.component.html

```diff
@@ -0,0 +1,2 @@
+<button type="button" class="btn btn-default navbar-btn"
+ (click) = "onClickSave($event)">Save</button>
```

Comment:
This would be better in one line, and keep the spacing like in `ng2-json-editor`

```
(click)="onClickSave($event)"
```